test/dynamo/test_higher_order_ops.py

```diff
@@ -288,7 +304,7 @@ def f(x, y):
         y = 0.5
         actual_graph = self._test_wrap_simple(
             f,
-            (x, y),
+            default_args_generator((x, y)),
```

Comment:
As written this just produces (x, y).  Ideally we would go through some of these (like this particular example) and change it to generate [(torch.randn(3, 1), 0.5), (torch.randn(3, 1), 0.4)] -- this way we'd test if we ended up specializing on a constant.